proposal.rst

```diff
@@ -317,48 +317,50 @@ Project Timeline
+      - make PR for these changes and getting it reviewed and completing user guide documentation simultaneously (2 days)
+  Week 6(5 JULY-11 JULY)
       - implement Square root global frequency IDF (1 day)
       - write test cases for Square root global frequency IDF (2 days)
-      - make PR for these changes and getting it reviewed and completing documentation simultaneously (1 day)
-  Week 6(5 JULY-11 JULY)
-      - get max-tf from Weight::Internal (2 days)
+      - make PR for these changes and getting it reviewed and completing user guide documentation simultaneously (1 day)
+      - get max-tf from Weight::Internal (1 day)
+  Week 7(12 JULY-18 JULY)
+      - get max-tf from Weight::Internal (1 day)
       - implement "max-norm" and "aug-norm" (1 day)
       - write test cases for max-tf (2 days)
-  Week 7(12 JULY-18 JULY)
-      - write test cases for "max-norm" and "aug-norm (3 days)
-      - make PR for these changes and getting it reviewed and completing documentation simultaneously(2 day)
+      - write test cases for "max-norm" and "aug-norm (1 day)
   Week 8(19 JULY-25 JULY)
-      - Add support for the new normalisations in Weight::create().This includes the changes to be made in Registry(3 days)
-      - make PR for these changes and getting it reviewed and completing documentation simultaneously (1 day)
-      - Rest of days are kept free to complete back log if any.
+      - write test cases for "max-norm" and "aug-norm (2 days)
+      - make PR for these changes and getting it reviewed and completing user guide documentation simultaneously(3 day)
   Week 9(26 JULY-1 AUG)[Evaluation 2]
       - submit evaluation.
+      - Add support for the new normalisations in Weight::create().This includes the changes to be made in Registry(3 days)
+      - make PR for these changes and getting it reviewed and completing user guide documentation simultaneously (1 day)
+  Week 10(2 AUG-8 AUG)
       - make changes in config_file.cc (2 days)
       - make changes in config_file.h (1 day)
       - make changes in trec_search.cc (2 days)
-  Week 10(2 AUG-8 AUG)
-      - write test cases for these changes (3 days)
-      - make PR for these changes and getting it reviewed and completing documentation simultaneously (2 days)
   Week 11(9 AUG-15 AUG)
-      - Compare effectiveness of different normalisations .
-      - make documentation for these results.
+      - make PR for these changes (2 days)
+      - Compare effectiveness of different normalisations. We will use FIRE database for this.
   Week 12(16 AUG-22 AUG)
-  This week is kept as buffer week.Also due to the COVID-19 outbreak in my country,Our exams have been rescheduled but the dates aren't decided.So there is a possibility that I might have exams during the coding period.But they wont last more than a week.If I have time,I can work for Stretch goals.
+      - make user guide documentation for these results.
+      - If I have time,I can work for Stretch goals.
```

Comment:
Exams during the coding period are something that happens, and unknown/uncertain dates due to COVID19 restrictions are clearly not under your control, but it's certainly good to note things like this as you had before, so do leave this in unless the uncertainty has since been resolved and the exams are definitely not going to happen during the coding period.